extra_geom/detectors.py

```diff
@@ -1014,6 +1014,90 @@ class DSSC_1MGeometry(DetectorGeometryBase):
         [0.5, 1, 1, 0.5, 0, 0]
     ])
 
+    @classmethod
+    def from_quad_positions(cls, quad_pos, *, unit=1e-3, asic_gap=None,
+                            panel_gap=None):
```

Comment:
I realised now that we are a bit inconsistant in the default values of asic and panel gap, sometimes it's `None` and sometime it's the default value.

Reply:
We are, yup. My take is that it should be None wherever `unit` affects how the gaps are interpreted (which I think is everywhere you can pass them), so you can specify `unit` without also having to specify the gaps. We should go through and make that consistent.